modules/ROOT/partials/configuration/extended_valid_elements.adoc

```diff
@@ -22,7 +21,6 @@ tinymce.init({
 
 Also see xref:valid_elements[`valid_elements`] and xref:invalid_elements[`invalid_elements`] for more configuration options.
```

Comment:
references in this line refer to things in other files, it's better to include the full ref like so:
```suggestion
Also see xref:configure/content-filtering.adoc#valid_elements[`valid_elements`] and xref:configure/content-filtering.adoc#invalid_elements[`invalid_elements`] for more configuration options.
```